bundlewrap/items/postgres_roles.py

```diff
@@ -17,14 +18,12 @@ def delete_role(node, role):
 
 
 def fix_role(node, role, attrs, create=False):
-    password = " PASSWORD '{}'".format(attrs['password_hash'])
-    sql = "{operation} ROLE \\\"{role}\\\" WITH LOGIN {superuser}SUPERUSER{password}".format(
-        operation="CREATE" if create else "ALTER",
-        password="" if attrs['password_hash'] is None else password,
-        role=role,
-        superuser="" if attrs['superuser'] is True else "NO",
-    )
-    node.run(f"psql -nqw -c \"{sql}\"", user="postgres")
+    if create:
+        create_sql = f"CREATE ROLE \\\"{role}\\\" WITH LOGIN {attrs['superuser']}SUPERUSER"
+        node.run(f"psql -nqw -c {quote(create_sql)}", user="postgres")
+
+    password_sql = f"UPDATE pg_authid SET rolpassword = '{attrs['password_hash']}' WHERE rolname = '{role}'"
```

Comment:
Der Fall `attrs['password_hash'] is None` wird hier nicht mehr bedacht.

Reply:
Nach meinem Verständnis was das vorher schon überflüssig, weil dank der Funktion `patch_attributes()` der `password_hash` in jedem Fall gesetzt ist.

Reply:
Nur, wenn auch ein Passwort gesetzt wurde.

Reply:
Wenn beides nicht gesetzt wurde, dann failt es hier: https://github.com/bundlewrap/bundlewrap/pull/772/files#diff-84f1936d9751760430f618f758c6ed889e625934e1342d4019cf102e9e793421R96